src/language_features/completion.rs

```diff
@@ -36,14 +37,35 @@ pub fn editor_completion(
         CompletionResponse::List(list) => list.items,
     };
     let re = Regex::new(r"(?P<c>[:|$])").unwrap();
+    let maxlen = items
+        .iter()
+        .map(|x| {
+            x.label.len()
+        }).max().unwrap_or(0);
+
     let items = items
         .into_iter()
         .map(|x| {
+            let mut doc: String = match &x.documentation {
+                None => "".to_string(),
+                Some(doc) => match doc {
+                    Documentation::String(st) => st.clone(),
+                    Documentation::MarkupContent(mup) => mup.value.clone(),
+                },
+            };
+            if let Some(d) = x.detail {
+                doc = d.clone() + "\n\n" + &doc;
```

Comment:
I think `format!` macro could be a perfect fit here: `doc = format!("{}\n\n{}", d, doc);`